FlattenBinaryTreeToLL.java

```diff
@@ -0,0 +1,36 @@
+Time Complexity - O(n*n)
+Space Complexity - O(n)

```

Comment:
Check time and space complexity 
For space, are you considering recursive stack?